app/identifiers/register/company/CompanyRegistrationNumberId.scala

```diff
@@ -17,7 +17,26 @@
 package identifiers.register.company
 
 import identifiers._
+import play.api.i18n.Messages
+import utils.UserAnswers
+import utils.checkyouranswers.{BooleanCYA, CheckYourAnswers, CheckYourAnswersCompany, StringCYA}
+import viewmodels.{AnswerRow, Link}
 
 case object CompanyRegistrationNumberId extends TypedIdentifier[String] {
+  self =>
   override def toString: String = "companyRegistrationNumber"
+
+  implicit def cya(implicit userAnswers: UserAnswers, messages: Messages): CheckYourAnswers[self.type] =
```

Comment:
implcit useranswers can be removed as it is already there in row

Reply:
done